### PR TITLE
Fixing LocalFileDetector choice when using non-local grid. Adding tes…

### PR DESCRIPTION
**Pull request by gabrielstar**
New test passed: http://10.40.234.40:8080/job/MrChecker-SDLC/job/CI_mrchecker-selenium-module/105/
All tests are not passing for a long time but this is not related to my change: http://10.40.234.40:8080/job/MrChecker-SDLC/job/CI_mrchecker-selenium-module/104/ . Here my  tests are also passing.

I am not too sure why there are so many formatting issues. I am using Allure formatter but perhaps those files followed some old format type ?

If that is an issue, can someone please contact me to look into this.

If it comes to my changes then instead of 

`static {
		configureFileDetector();
	}`
.... 
we can do a one-liner
`private static FileDetector		defaultFileDetector	= new LocalFileDetector();`

with some small performance cost. Then files will be copied over to temp locations also for connections to grid when it is on the same host.

### CODE REVIEW

File: mrchecker-framework-modules/mrchecker-selenium-module/src/main/java/com/capgemini/mrchecker/selenium/core/newDrivers/NewRemoteWebElement.java [hunk 1]
```diff
@@ -1,63 +1,33 @@
...
-import org.openqa.selenium.By;
-import org.openqa.selenium.ElementNotVisibleException;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.StaleElementReferenceException;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.remote.FileDetector;
-import org.openqa.selenium.remote.RemoteWebDriver;
-import org.openqa.selenium.remote.RemoteWebElement;
-import org.openqa.selenium.remote.UselessFileDetector;
+import org.openqa.selenium.*;
```
Comment (przwojtkow) [inline mrchecker-framework-modules/mrchecker-selenium-module/src/main/java/com/capgemini/mrchecker/selenium/core/newDrivers/NewRemoteWebElement.java:10]: Please check if we need to import whole package. By default formatter should be set to 5 classes to import whole pkg, but some issues faced before.

File: mrchecker-framework-modules/mrchecker-selenium-module/src/main/java/com/capgemini/mrchecker/selenium/core/newDrivers/NewRemoteWebElement.java [hunk 2]
```diff
@@ -1,63 +1,33 @@
...
+import org.openqa.selenium.remote.*;
```
Comment (przwojtkow) [inline mrchecker-framework-modules/mrchecker-selenium-module/src/main/java/com/capgemini/mrchecker/selenium/core/newDrivers/NewRemoteWebElement.java:11]: Same here.

**Review by przwojtkow — COMMENTED** [review]
Please share with issue description.

**Comment by michalbabiarz** [comment]
PR is broken. I'm going to reopen.

**Pull request closed without merging**